plugins/git-prompt/gitstatus.py

```diff
@@ -17,9 +17,9 @@ def get_tagname_or_hash():
     if m:
         hash_ = output[:m.start()-1]
     # get tagname
-    m = re.search('tag: .*[,\)]', output)
+    m = re.search('tag: .*?(?=(, |\)))', output)
```

Comment:
Why are you adding a look ahead here?
Isn't that worse performance than just using `.*[,\) ]' ?